### DISCUSSION TITLE
remove dashboard-nvim plugin

### DISCUSSION THREAD

**Discussion by ghost** [LazyVim/LazyVim#2017] [Q&A]
I'm looking for a way to delete the dashboard on lazyvim template, because I don't like using this plugin

**Comment by dpetka2001** [comment]
Have you read the [docs](https://www.lazyvim.org/configuration/plugins#-disabling-plugins)?

**Reply by dpetka2001** [reply]
Also if you'd like to have the stock Neovim screen when you start, take a look at #1946 for how to achieve that.

**Reply by ghost** [reply]
LOL, looks like I missed an important part, thank you
